src/controllers/trainee/Controller.ts

```diff
@@ -0,0 +1,85 @@
+import { Request, Response } from 'express';
+import * as bcrypt from 'bcrypt';
+import UserRepository from '../../repositories/user/UserRepository';
+import SystemResponse from '../../libs/SystemResponse';
+class TraineeController {
+    static instance;
+    private userRepository: UserRepository = new UserRepository();
+    static getInstance() {
+        if (TraineeController.instance)
+         return TraineeController;
+        TraineeController.instance = new TraineeController();
+          return TraineeController.instance;
+    }
+    create = async (req: Request, res: Response) => {
+        console.log('::::CREATE TRAINEE:::::');
+        try {
+            const traineeData = req.body;
+            const hash = await bcrypt.hash(traineeData.password, 10);
+            const trainee = await this.userRepository.create({ ...traineeData, password: hash});
+                return SystemResponse.success(res, trainee, 'Trainee Added Successfully');
+        }
+        catch (error) {
+                return SystemResponse.error(res, error, 'Trainee Added UnSuccessfull');
+            }
+    }
+    update = async (req: Request, res: Response) => {
+        console.log('::::UPDATE TRAINEE:::::');
+        try {
+            const traineeData = req.body;
+            const traineeId = traineeData.id;
+        const trainee = await this.userRepository.update1(traineeData.id, traineeData.dataToUpdate, traineeId);
```

Comment:
Here first and thord arguments are same and will have same value then why do we need to pass same value a different argumemts